package.json

```diff
@@ -122,7 +122,7 @@
     "react": "16.8.6",
     "react-dom": "16.8.6",
     "react-native": "0.60.4",
-    "react-native-bpk-appearance": "^1.3.1",
+    "react-native-bpk-appearance": "file:/./packages/react-native-bpk-appearance",
```

Comment:
Because is a peer dependency now this is so we can make changes to the package and use the latest version in dev like we do for all other packages

Reply:
I guess I'll add a comment for now because we still want to move RN to a single package, which will change this

Reply:
Ah yeah of course :)